Cargo.toml

```diff
@@ -14,6 +14,9 @@ members = [
     "contract-sdk/storage",
     "contract-sdk-macros",
 
+    # Runtime Client
+    "client-sdk/rust",
```

Comment:
Don't forget to also update the docs jobs to generate documentation for this crate and add a reference link to `docs/toc.md`.